generate/generate_nircam.py

```diff
@@ -318,100 +342,126 @@
 aperture_collection = pysiaf.ApertureCollection(aperture_dict)
 
 emulate_delivery = True
-emulate_delivery = False
+# emulate_delivery = False
 
 if emulate_delivery:
     pre_delivery_dir = os.path.join(JWST_DELIVERY_DATA_ROOT, instrument)
     if not os.path.isdir(pre_delivery_dir):
         os.makedirs(pre_delivery_dir)
 
     # write the SIAF files to disk
-    filenames = pysiaf.iando.write.write_jwst_siaf(aperture_collection, basepath=pre_delivery_dir, file_format=['xml', 'xlsx'])
+    filenames = pysiaf.iando.write.write_jwst_siaf(aperture_collection, basepath=pre_delivery_dir, file_format=['xml'], label='jwstsiaf-129')
 
-    pre_delivery_siaf = pysiaf.Siaf(instrument, basepath=pre_delivery_dir)
+    # 1/0
+    # pre_delivery_siaf = pysiaf.Siaf(instrument, basepath=pre_delivery_dir)
+    pre_delivery_siaf = pysiaf.Siaf(instrument, filename=filenames[0])
 
     # compare new SIAF with PRD version
-    ref_siaf = pysiaf.Siaf(instrument)
-    compare.compare_siaf(pre_delivery_siaf, reference_siaf_input=ref_siaf, fractional_tolerance=1e-6, report_dir=pre_delivery_dir, tags={'reference': pysiaf.JWST_PRD_VERSION, 'comparison': 'pre_delivery'})
+    # ref_siaf = pysiaf.Siaf(instrument)
```

Comment:
Maybe want to try and find a way to move the checks on lines 360, 362, and 366 somewhere more appropriate like a separate function? Or if that isn't needed, just delete them?